stable/kube-state-metrics/README.md

```diff
@@ -0,0 +1,21 @@
+# kube-state-metrics Helm Chart
+
+* Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install stable/kube-state-metrics
+```
+
+## Configuration
+
+| Parameter          | Description                         | Default                                      |
+|--------------------|-------------------------------------|----------------------------------------------|
+| `image.repository` | The image repository to pull from   | gcr.io/google_containers/kube-state-metrics  |
+| `image.tag`        | The image tag to pull from          | v0.4.1                                       |
+| `image.pullPolicy` | Image pull policy                   | IfNotPresent                                 |
+| `service.port`     | The port of the container           | 8080                                         |
```

Comment:
Ordinarily this should be collapsed to something like `servicePort` since it's the only value left in the map. But I expect we'll be adding more, so let's leave it as-is.